demo/crypto-perf-dpdk-pod-requesting-qat.yaml

```diff
@@ -16,12 +16,12 @@ spec:
       requests:
         cpu: "3"
         memory: "128Mi"
-        intel.com/qat: '4'
+        qat/generic: '4'
```

Comment:
I guess this should be "qat.intel.com/generic".

Reply:
Well, I noticed that in the code here : https://github.com/intel/intel-device-plugins-for-kubernetes/blob/master/cmd/qat_plugin/qat_plugin.go#L49  which was changed in the PR https://github.com/intel/intel-device-plugins-for-kubernetes/commit/b6894b8195d253f2203b7da9ee47baf49162918f   that the namespace is initialized as qat. Would you like me to change the namespace to qat.intel.com?

Reply:
Oops. Yes, the namespace should be "qat.intel.com" in qat_plugin.go, not "qat".